src/app/technical-record/vehicle-summary/vehicle-summary-hgv/vehicle-summary-hgv.component.html

```diff
@@ -0,0 +1,91 @@
+<dl class="govuk-summary-list">
+  <div class="govuk-summary-list__row">
+    <dt class="govuk-summary-list__key">
+      Speed limiter exempt
+    </dt>
+    <dd id="test-speedLimiterMrk" class="govuk-summary-list__value">
```

Comment:
once again confused here.
I only had off road vehicle and number of wheel driven shown for HGV and we seem to be rendering more form fields now?